internal/cmd/root/root.go

```diff
@@ -129,36 +128,53 @@ func Execute() {
 
 	var cmd *cobra.Command
 	if cmd, err = rootCmd.ExecuteC(); err != nil {
-		// We only want to print the usage message if an ErrIncorrectUsage
-		// was returned or it's an error produced by cobra which was
-		// caused by incorrect usage
+		// Error types that need special handling
 		var usageErr *cmdutils.IncorrectUsageError
+		var couldBeSandboxError *cmdutils.CouldBeSandboxError
+		var signalErr *cmdutils.SignalError
+		var silentErr *cmdutils.SilentError
+
 		if errors.As(err, &usageErr) ||
 			strings.HasPrefix(err.Error(), "unknown command") ||
 			regexp.MustCompile(`(accepts|requires).*arg\(s\)`).MatchString(err.Error()) {
-
-			// Ensure that there is an extra newline between the error
-			// and the usage message
-			if !strings.HasSuffix(err.Error(), "\n") {
-				_, _ = fmt.Fprintln(cmd.ErrOrStderr())
-			}
-
 			// Make cmd.Help() print to stderr
 			cmd.SetOut(cmd.ErrOrStderr())
+
 			// Print the usage message of the command. We use cmd.Help()
 			// here instead of cmd.UsageString() because the latter
 			// doesn't include the long description.
 			_ = cmd.Help()
+
+			// Add "Invalid Usage" in front of error message
+			err = errors.Wrap(err, "Invalid usage")
```

Comment:
We don't want to attach a stack trace here
```suggestion
			err = errors.WithMessage(err, "Invalid usage")
```